shell/auto_test/tests/org/organization-manage.spec.ts

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2021 Terminus, Inc.
+//
+// This program is free software: you can use, redistribute, and/or modify
+// it under the terms of the GNU Affero General Public License, version 3
+// or later ("AGPL"), as published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+// FITNESS FOR A PARTICULAR PURPOSE.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+import { Role, test, expect } from '../../fixtures';
+import Base from '../pages/base';
+const title = 'terminus' + Date.now();
+const testData = {
+  image: 'app/images/Erda.png',
+};
+
+Role('Manager', () => {
+  test('management-center', async ({ page, expectExist, goTo }) => {
```

Comment:
Test case `management-center` is just create project ?

Reply:
modified, Set the rollback point, click the project to jump to the project market